### PR TITLE
CT-2141 Removed last vestiges of DACU from front end

### PR DESCRIPTION
**Pull request by starswan**

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -370,7 +370,7 @@ en:
         message: Full request
         postal_address: Address
         flag_for_disclosure_specialists:
-          Clearance required by DACU Disclosure?
+          Clearance required by Disclosure BMT?
```
Comment (beccapearce) [inline config/locales/en.yml]: This should just be disclosure not disclosure bmt

File: config/locales/state.en.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@ en:
     awaiting_responder_email: Waiting to be accepted
     awaiting_dispatch: Ready to send
     drafting: Draft in progress
-    pending_dacu_clearance: Pending clearance
+    pending_dacu_clearance: Pending clearance - Disclosure
```
Comment (beccapearce) [inline config/locales/state.en.yml]: Do we want to change this? It will change it on all the case listings

**Review by beccapearce — CHANGES_REQUESTED** [review]
Replace Disclosure with disclosure BMT also a question about changing the state file

File: config/locales/state.en.yml (hunk 2 above)
Comment (mseedat-moj) [reply config/locales/state.en.yml]: Reverted so available on rebase to master

**Review by beccapearce — APPROVED** [review]

**Pull request merged**